src/ewatercycle/base/forcing.py

```diff
@@ -299,6 +309,50 @@ def variables(self) -> tuple[str, ...]:
         """
         return tuple(self.filenames)
 
+    def get_shape_area(self) -> float:
+        """Return the area of the shapefile in m2.
+        """
+        if self.shape == None:
+            raise ("Shapefile not specified")
+        shape = fiona.open(self.shape)
+        poly = [shapely.geometry.shape(p["geometry"]) for p in shape][0]
+        geod = Geod(ellps="WGS84")
+        poly_area, _ = geod.geometry_area_perimeter(poly)
+        catchment_area_m2 = abs(poly_area)
+        return catchment_area_m2
+
+
+    def plot_shape(self) -> matplotlib.axes._axes.Axes:
+        """Make a plot of the shapefile on a map of the world.
+        The plot is padded with 10% of the north-south extend,
+        or the east-west extend (which ever is smallest).
+
+        the axis object is ter
+        """
+
+        if self.shape == None:
+            raise ("Shapefile not specified")
+
+        shape = fiona.open(self.shape)
+        poly = [shapely.geometry.shape(p["geometry"]) for p in shape][0]
+        w, s, e, n = poly.bounds  # different order than set_extent expects
+        
+        # 10 % of the minimum of either the west-east extend or the north-south extend is used as
+        # padding around the shape.
+        pad = min(abs(w - e), abs(n - s)) * 0.1 
+        
+        plt.figure(figsize=(8, 5))
+        ax = plt.axes(projection=ccrs.PlateCarree())
+        ax.add_geometries(
+            poly, crs=ccrs.PlateCarree(), facecolor='#f5b41d', edgecolor='k', alpha=0.8,
+        )
+        ax.add_feature(cfeature.COASTLINE, linewidth=1)
+        ax.add_feature(cfeature.RIVERS, linewidth=1)
+        ax.add_feature(cfeature.OCEAN, edgecolor="none", facecolor="#4287f5")
+        
+        ax.set_extent((w-pad, e+pad, s-pad, n+pad), crs=ccrs.PlateCarree())
```

Comment:
```suggestion
        ax = plt.axes(projection=ccrs.PlateCarree())
        ax.add_geometries(  # type: ignore
            poly, crs=ccrs.PlateCarree(), facecolor='#f5b41d', edgecolor='k', alpha=0.8,
        )
        ax.add_feature(cfeature.COASTLINE, linewidth=1)  # type: ignore
        ax.add_feature(cfeature.RIVERS, linewidth=1)  # type: ignore
        ax.add_feature(cfeature.OCEAN, edgecolor="none", facecolor="#4287f5")  # type: ignore
        
        ax.set_extent((w-pad, e+pad, s-pad, n+pad), crs=ccrs.PlateCarree())  # type: ignore
```
Typing doesn't work well with matplotlib... This ignores the typing on the offending lines, and should make the tests